components/automate-backend-elasticsidecar/habitat/automate-backend-elasticsidecar/automate-backend-elasticsidecar.gemspec

```diff
@@ -0,0 +1,26 @@
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require_relative "lib/automate/backend/elasticsidecar/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "automate-backend-elasticsidecar"
+  spec.version       = Automate::Backend::Elasticsidecar::VERSION
+  spec.authors       = ["Faizan Fulara"]
+  spec.email         = ["ffulara@progress.com"]
+  spec.description   = %q{Package to control backend-elasticsidecar}
+  spec.summary       = spec.description
+  spec.licenses      = "Apache-2.0"
+
+  spec.require_paths = ["lib"]
+
+  spec.required_ruby_version = ">= 2.4.0"
```

Comment:
This is an EOL ruby release. We shouldn't support it.

Reply:
Okay , will upgrade the support to 2.6 and greater